demo/src/timetables.py

```diff
@@ -139,16 +145,20 @@ def create_cliquet_timetable(monthend_datetimes, spot, trial, params):
     m_exp = 12
     fix_dates = monthend_datetimes[trial + m_per : trial + m_exp + 1 : m_per]
 
+    # Reverse cap_floor values and divide by 100
+    local_cap, local_floor = [val / 100 for val in reversed(params.get("cap_floor", [0.05, -0.05]))]
+
```

Comment:
or simpler.

```
local_floor, local_cap = [val / 100 for val in params.get("cap_floor", [0.05, -0.05])]

```

also, to reduce confusion, call the parameter "floor_cap" instead of "cap_floor"